app/models/family.rb

```diff
@@ -1,3 +1,5 @@
 class Family < ApplicationRecord
   validates_presence_of :last_name, :capability
+
+  scope :by_last_name, -> { reorder("lower(last_name)") }
```

Comment:
Style guide prefers using the `def self.by_last_name` syntax.

Also a question: why `reorder`? and what is the need for `lower`?
Could this work?

```
def self.by_last_name
  order(last_name: :desc)
end
```

Reply:
Also add a spec for this scope...

Reply:
It was not working properly and reorder is a leftover from my attempt 😅(I will remove it).
The problem is that it is case sensitive so I lower_cased the last name to make sure the order is correct. 
Done with fixup in sha `fc216f6`

Reply:
Spec done in sha `940d912`